src/sdc11073/wsdiscovery/__init__.py

```diff
@@ -0,0 +1,14 @@
+# noqa: D104
+from .common import MULTICAST_PORT
+from .wsdimpl import (

```

Comment:
modules that should not be fully publicly available, because they are exported here should have a `_` prefix